javav2/example_code/medicalimaging/src/main/java/com/example/medicalimaging/UpdateImageSetMetadata.java

```diff
@@ -45,123 +46,155 @@ public static void main(String[] args) {
+                                .removableAttributes(SdkBytes.fromByteBuffer(
+                                        ByteBuffer.wrap(removeAttributes
+                                                .getBytes(StandardCharsets.UTF_8))))
+                                .build())
+                        .build();
+
+                updateMedicalImageSetMetadata(medicalImagingClient, datastoreId, imagesetId,
+                        versionid, metadataRemoveUpdates, force);
+                // snippet-end:[medicalimaging.java2.update_image_set_metadata.remove_attributes]
+            } else if (updateType.equals("remove_instance")) {
+                // Remove an instance.
+                // snippet-start:[medicalimaging.java2.update_image_set_metadata.remove_instance]
+                final String removeInstance = """
+                        {
+                          "SchemaVersion": 1.1,
+                          "Study": {
+                            "Series": {
+                              "1.1.1.1.1.1.12345.123456789012.123.12345678901234.1": {
+                                "Instances": {
+                                  "1.1.1.1.1.1.12345.123456789012.123.12345678901234.1": {}
+                                }
+                              }
+                            }
+                          }
+                        }      
+                        """;
+                MetadataUpdates metadataRemoveUpdates = MetadataUpdates.builder()
+                        .dicomUpdates(DICOMUpdates.builder()
+                                .removableAttributes(SdkBytes.fromByteBuffer(
+                                        ByteBuffer.wrap(removeInstance
+                                                .getBytes(StandardCharsets.UTF_8))))
+                                .build())
+                        .build();
+
+                updateMedicalImageSetMetadata(medicalImagingClient, datastoreId, imagesetId,
+                        versionid, metadataRemoveUpdates, force);
+                // snippet-end:[medicalimaging.java2.update_image_set_metadata.remove_instance]
+            } else if (updateType.equals("revert")) {
+                // snippet-start:[medicalimaging.java2.update_image_set_metadata.revert]
+                // In this case, revert to previous version.
+                String revertVersionId = Integer.toString(Integer.parseInt(versionid) - 1);
+                MetadataUpdates metadataRemoveUpdates = MetadataUpdates.builder()
+                        .revertToVersionId(revertVersionId)
+                        .build();
+                updateMedicalImageSetMetadata(medicalImagingClient, datastoreId, imagesetId,
+                        versionid, metadataRemoveUpdates, force);
+                // snippet-end:[medicalimaging.java2.update_image_set_metadata.revert]
+            }
+        } catch (MedicalImagingException e) {
+            System.err.println(e.awsErrorDetails().errorMessage());
```

Comment:
Log and forget all exceptions is not a great practice.
If possible catch a specific exception and take a specific action.